ramiro_serra/dif_driver_investigation.md

```diff
@@ -0,0 +1,53 @@
+# Differential wheeled robots
+
+A differential wheeled robot is a mobile robot whose movement is based on two separately driven wheels placed on either side of the robot body. It can thus change its direction by varying the relative rate of rotation of its wheels and hence does not require an additional steering motion.
+
+![Robot\](https://www.google.com/url?sa=i&rct=j&q=&esrc=s&source=images&cd=&ved=2ahUKEwjQgfuB67TkAhUbIrkGHcxLCWUQjRx6BAgBEAQ&url=http://planning.cs.uiuc.edu/node659.html&psig=AOvVaw2lkNnIE3pIZqVMvjR70tXw&ust=1567606381458775 )](http://planning.cs.uiuc.edu/img5518.gif)
+
+###### Differential wheeled robot
```

Comment:
Use [headings levels](https://www.markdownguide.org/basic-syntax/#headings) in order.